src/compiler/crystal/tools/doc/project_info.cr

```diff
@@ -133,44 +130,35 @@ module Crystal::Doc
 
     def self.git_clean?
       # Use git to determine if index and working directory are clean
-      io = IO::Memory.new
-      status = Process.run("git", ["status", "--porcelain"], output: io)
+      capture = Crystal::Git.git_capture(["status", "--porcelain"]) { return }
+
       # If clean, output of `git status --porcelain` is empty. Still need to check
```

Comment:
I guess the comment should be placed within the `{ return }` block, right?

Reply:
No, why?

Reply:
git_capture checks the status code. The `return unless status.success?` is gone.

Reply:
Oh, sry I was confused. You're right.